BUILD

```diff
@@ -95,6 +95,88 @@ java_plugin(
     ],
 )
 
+java_library(
+    name = "springboot",
+    visibility = [
+        "//visibility:public",
+    ],
+    exports = [
+        "@maven//:javax_servlet_javax_servlet_api",
+        "@maven//:javax_validation_validation_api",
+        "@maven//:org_springframework_boot_spring_boot",
+        "@maven//:org_springframework_boot_spring_boot_actuator",
+        "@maven//:org_springframework_boot_spring_boot_actuator_autoconfigure",
+        "@maven//:org_springframework_boot_spring_boot_autoconfigure",
+        "@maven//:org_springframework_boot_spring_boot_loader",
+        "@maven//:org_springframework_boot_spring_boot_starter",
+        "@maven//:org_springframework_boot_spring_boot_starter_actuator",
+        "@maven//:org_springframework_boot_spring_boot_starter_jetty",
+        "@maven//:org_springframework_boot_spring_boot_starter_web",
+        "@maven//:org_springframework_spring_beans",
+        "@maven//:org_springframework_spring_context",
+        "@maven//:org_springframework_spring_core",
+        "@maven//:org_springframework_spring_web",
+        "@maven//:org_springframework_spring_webmvc",
+    ],
+)
+
+java_library(
+    name = "junit",
+    visibility = [
+        "//visibility:public",
+    ],
+    exports = [
+        "@maven//:com_jayway_jsonpath_json_path",
+        "@maven//:org_hamcrest_hamcrest",
+        "@maven//:org_hamcrest_hamcrest_library",
+        "@maven//:org_junit_jupiter_junit_jupiter",
+        "@maven//:org_junit_jupiter_junit_jupiter_api",
+        "@maven//:org_junit_jupiter_junit_jupiter_engine",
+        "@maven//:org_junit_jupiter_junit_jupiter_params",
+        "@maven//:org_mockito_mockito_core",
+    ],
+)
+
+java_library(
+    name = "springboot_websocket",
+    visibility = [
+        "//visibility:public",
+    ],
+    exports = [
+        "@maven//:org_springframework_boot_spring_boot_starter_websocket",
+        "@maven//:org_springframework_spring_messaging",
+        "@maven//:org_springframework_spring_websocket",
+    ],
+)
+
+java_library(
+    name = "springboot_security",
+    visibility = [
+        "//visibility:public",
+    ],
+    exports = [
+        "@maven//:org_springframework_boot_spring_boot_starter_security",
+        "@maven//:org_springframework_security_spring_security_config",
+        "@maven//:org_springframework_security_spring_security_core",
+        "@maven//:org_springframework_security_spring_security_web",
+    ],
+)
+
+java_library(
+    name = "springboot_test",
+    visibility = [
+        "//visibility:public",
+    ],
+    exports = [
+        ":junit",
```

Comment:
not a big fan of this, but works well in practise... I'm thinking if we introduce `backend/BUILD` we can have an umbrella dep there